recipes/qtils/all/conanfile.py

```diff
@@ -0,0 +1,43 @@
+import os
+from conan import ConanFile
+from conan.tools.cmake import CMakeToolchain, CMakeDeps, cmake_layout
+from conan.tools.files import copy,get
+
+required_conan_version = ">=2.0.5"
```

Comment:
We still need to support Conan 1 for now.
```suggestion
required_conan_version = ">=1.53.0"
```